staff/aitor-parra/precourse/bingo/index.js

```diff
@@ -0,0 +1,130 @@
+//BINGO versión 15 números no aleatorios.
+
+// EL CÓDIGO SE EJECUTA CON LA FUNCIÓN ->   Bingo() 
+
+
+var playerCounter = 0;
+var playerS = []
+var firstLineEnd = false;
+var secondLineEnd = false;
+var thirdLineEnd = false;
+var bingoLine = false;
```

Comment:
As in Bingo game we only say "Linea" once, I suggest creating only one variable, f.i var lineaEnd = false;
Then, if one linea is finished, lineaEnd = true. If(lineaEnd) {only search Bingo}

Reply:
I see you point. "Line" should only be said once.
I fixed it adding an IF CONDITION that checks the variable that check if each line is completed, this way the program will only say "Line" once. I decide to mantain the three variables because is useful to check if all three are completed to say "Bingo!"
Thank you so much for your review and comments to let me improve my code!